source/gameengine/Ketsji/KX_GameObject.cpp

```diff
@@ -236,7 +240,10 @@ void KX_GameObject::SetName(const char *name)
 
 PHY_IPhysicsController* KX_GameObject::GetPhysicsController()
 {
-	return m_pPhysicsController;
+	if (m_pPhysicsController && !m_suspendedPhysics) {
```

Comment:
It happens at least for mass and as almost all attributes are set like mass, I guess this is the same